stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -81,12 +93,15 @@ public function isAbsolute(string path) returns boolean|Error {
     }
 }
 
-# Retrieves the base name of the file from the provided location.
-# The last element of path.
+# Retrieves the base name of the file from the provided location,
+# which is the last element of the path.
 # Trailing path separators are removed before extracting the last element.
+# ```ballerina
+#  string|filepath:Error name = filepath:filename("/A/B/C.txt");
+# ```
 #
-# + path - String value of file path.
-# + return - Returns the name of the file
+# + path - String value of file path
+# + return - The name of the file
```

Comment:
Again, need to document the error as well

Reply:
Corrected.